### PR TITLE
MBD/docs-audit

### PR DESCRIPTION
**Pull request by marabd**
made several audits to the documentation doing a little editing, and adding or removing any necessary props.

### CODE REVIEW

File: docs/components/IconDocs.js [hunk 1]
```diff
@@ -80,6 +81,9 @@ class IconDocs extends React.Component {
           />
         </div>
 
+        <h5>elementProps <label>Object</label></h5>
+        <p>Pass props directly to the <Code>icon</Code> element. ie. focusable, etc.</p>
```
Comment (cerinman) [inline docs/components/IconDocs.js]: I'm not sure how we worded this for other components that have `elementProps` but this seems a bit general and slightly miss leading as to the intent of elementProps.

This seems to be more in line with the spirit of what elementProps is.

`Pass native HTML attributes for SVG elements directly to the icon's wrapping SVG element.  ie. height, width, etc.`

Comment (marabd) [reply docs/components/IconDocs.js]: thanks, I appreciate this more thorough description. updated!

File: docs/components/Components.js [hunk 2]
```diff
@@ -29,6 +29,10 @@ class Components extends React.Component {
                   value: 'button-group',
                   displayValue: 'Button Group'
                 },
+                {
+                  value: 'container-column-row',
+                  displayValue: 'Grid Contain, Column & Row'
```
[the remark this answers is not in the document]
Comment (marabd) [reply docs/components/Components.js]: yep. whoops. fixing

File: docs/components/PaginationButtonsDocs.js [hunk 3]
```diff
@@ -50,6 +51,10 @@ class PaginationButtonsDocs extends React.Component {
         <h5>totalPages <label>Number</label></h5>
         <p><em>(required)</em> The total number of pages.</p>
 
+        <h5>type <label>String</label></h5>
+        <p>Default: <Code>'primary'</Code></p>
+        <p>The sets the button type.</p>
```
Comment (cerinman) [inline docs/components/PaginationButtonsDocs.js]: weird grammar here.

Comment (marabd) [reply docs/components/PaginationButtonsDocs.js]: good catch, I never ended up finishing that one. just updated

File: docs/components/TabsDocs.js [hunk 4]
```diff
@@ -50,17 +50,24 @@ class TabsDocs extends React.Component {
...
         <p>Styles for the active tab.</p>
 
-        <h5>alignment<label>'left' or 'center'</label></h5>
+        <h5>alignment <label>'left' or 'center'</label></h5>
         <p>Default: 'left'</p>
         <p>Tab alignment.</p>
 
+        <h5>isActive <label>Boolean</label></h5>
+        <p>Default: 'false'</p>
+        <p>If set to true, allows for "activeTabeStyles" to apply to the active tab.</p>
```
Comment (cerinman) [inline docs/components/TabsDocs.js]: Typo here. `activeTabsStyles`

Comment (marabd) [reply docs/components/TabsDocs.js]: good catch. updated!

**Comment by cerinman** [comment]
I love you so much for doing this Mara....

giphy3
 :shipit:

**Pull request merged**